openedx/core/djangoapps/xblock/rest_api/views.py

```diff
@@ -102,7 +102,7 @@ def get_handler_url(request, usage_key_str, handler_name):
 # and https://github.com/edx/XBlock/pull/383 for context.
 @csrf_exempt
 @xframe_options_exempt
-def xblock_handler(request, user_id, secure_token, usage_key_str, handler_name, suffix):
+def xblock_handler(request, user_id, secure_token, usage_key_str, handler_name, suffix=None):
```

Comment:
According to [Django 3.0 release notes](https://docs.djangoproject.com/en/3.2/releases/3.0/)
> RegexPattern, used by re_path(), no longer returns keyword arguments with None values to be passed to the view for the optional named groups that are missing.